eng/scripts/Language-Settings.ps1

```diff
@@ -220,19 +220,27 @@ function Find-javascript-Artifacts-For-Apireview($artifactDir, $packageName = ""
   [regex]$pattern = "azure-"
   $pkgName = $pattern.replace($packageName, "", 1)
   $packageDir = Join-Path $artifactDir $pkgName "temp"
-  Write-Host "Searching for *.api.json in path $($packageDir)"
-  $files = Get-ChildItem "${packageDir}" | Where-Object -FilterScript { $_.Name.EndsWith(".api.json") }
-  if (!$files)
+  if (Test-Path $packageDir)
   {
-    Write-Host "$($packageDir) does not have api review json for package"
-    return $null
+    Write-Host "Searching for *.api.json in path $($packageDir)"
+    $files = Get-ChildItem "${packageDir}" | Where-Object -FilterScript { $_.Name.EndsWith(".api.json") }
+    if (!$files)
+    {
+      Write-Host "$($packageDir) does not have api review json for package"
+      return $null
+    }
+    elseif ($files.Count -ne 1)
+    {
+      Write-Host "$($packageDir) should contain only one api review for $($packageName)"
+      Write-Host "No of Packages $($files.Count)"
+      return $null
+    }
   }
-  elseif ($files.Count -ne 1)
+  else
   {
-    Write-Host "$($packageDir) should contain only one api review for $($packageName)"
-    Write-Host "No of Packages $($files.Count)"
+    Write-Host "$($pkgName) does not have api review json"
```

Comment:
What if anything should folks do if they hit this issue?

Reply:
Log an issue to get the api review file in.

We have #13387 that should fix this up across the repo. cc @KarishmaGhiya 
We have the code generator now using the api extractor, so any newly generated package should have this. 
Anybody using our template project as a reference should also get it

Reply:
I mainly asked the question here as I think we need to improve the message here to let folks know what they should do if they hit this.

Reply:
Agreed. How about

> THIS IS BAD! LOG A GITHUB ISSUE FOR THIS NOW!!!!

:)

Reply:
Updated log message

Reply:
In addition to this change, I will check if we can update client type for util package to different type after March release.  I will create and issue for tracking.There is an alternate thread on this discussion.